basics/create-a-solver/create-a-solver.jmd

```diff
@@ -0,0 +1,205 @@
+---
+title: Creating a JSO compatible solver
+author: João Okimoto
+---
+
+![](../../jso.png)
+
+```julia
+using Pkg
+pkg"activate ."
+if false
+ pkg"add NLPModels"
+ pkg"add SolverTools"
+ pkg"add SolverBenchmark"
+ pkg"add Plots"
+ pkg"add LinearAlgebra"
+ pkg"add JSOSolvers"
+end
+pkg"instantiate"
+pkg"status"
+```
+
+This tutorial covers the basics on how to create a solver compatible with the
+JuliaSmoothOptimizers format and do basic benchmarking.
+Let's start off with a simple implementation of Newton's method:
+```julia
```

Comment:
Before the code, show the algorithm, like this:

1. Do this
2. While that
	2.1 Do this
	2.2 Do this
3. Do this